clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/config/GoogleInfrastructureProviderConfig.groovy

```diff
@@ -130,6 +130,11 @@ class GoogleInfrastructureProviderConfig {
                                                                  objectMapper,
                                                                  region,
                                                                  registry)
+          newlyAddedAgents << new GoogleRegionalServerGroupCachingAgent(googleConfiguration.googleApplicationName(),
+                                                                        credentials,
+                                                                        objectMapper,
+                                                                        region,
+                                                                        registry)
           newlyAddedAgents << new GoogleServerGroupCachingAgent(googleConfiguration.googleApplicationName(),
```

Comment:
Is it worth it to rename `GoogleServerGroupCachingAgent` to `GoogleZonalServerGroupCachingAgent`?